docs/nextflow/reduce-api-calls.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Reducing Wave API calls
```

Comment:
I would change this title to something like  `Wave Freeze` and have the reduction of API calls be a secondary effect.

The benefit of wave freeze is enhanced security , availability and reliability by reducing the need for wave as part of a pipeline build, acting as a request proxy.